DNN Platform/DotNetNuke.Abstractions/Portals/IPortalInfo.cs

```diff
@@ -0,0 +1,310 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Portals
+{
+    using System;
+
+    /// <summary>
+    /// The portal info.
+    /// </summary>
+    public interface IPortalInfo
+    {
+        /// <summary>
+        /// Gets or sets the footer text as specified in the Portal settings.
+        /// </summary>
+        /// <value>Footer text of the portal.</value>
+        /// <returns>Returns the the footer text of the portal.</returns>
+        /// <remarks>
+        /// <example>This show the usage of the <c>FooterText</c> property
+        /// <code lang="vbnet">
+        /// txtFooterText.Text = objPortal.FooterText
+        /// </code>
+        /// </example>
+        /// </remarks>
+        string FooterText { get; set; }
+
+        /// <summary>
+        /// Gets or sets home directory of the portal (logical path).
+        /// </summary>
+        /// <value>Portal home directory.</value>
+        string HomeDirectory { get; set; }
+
+        /// <summary>
+        /// Gets home System (local) directory of the portal (logical path).
+        /// </summary>
+        /// <value>Portal home system directory.</value>
+        string HomeSystemDirectory { get; }
+
+        /// <summary>
+        /// Gets fysical path on disk of the home directory of the portal.
+        /// </summary>
+        /// <value>Physical path on disk of the home directory of the portal.</value>
+        string HomeDirectoryMapPath { get; }
+
+        /// <summary>
+        /// Gets fysical path on disk of the home directory of the portal.
+        /// </summary>
+        /// <value>Physical path on disk of the home directory of the portal.</value>
+        string HomeSystemDirectoryMapPath { get; }
+
+        /// <summary>
+        /// Gets or sets userID of the user who is the admininistrator of the portal.
+        /// </summary>
+        /// <value>UserId of the user who is the portal admin.</value>
+        int AdministratorId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the RoleId of the Security Role of the Administrators group of the portal.
+        /// </summary>
+        /// <value>RoleId of de Administrators Security Role.</value>
+        int AdministratorRoleId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabId at which admin tasks start.
+        /// </summary>
+        /// <value>TabID of admin tasks.</value>
+        int AdminTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets image (bitmap) file that is used as background for the portal.
+        /// </summary>
+        /// <value>Name of the file that is used as background.</value>
+        string BackgroundFile { get; set; }
```

Comment:
@mitchelsellers this should be deprecated, right?  Which, I guess, means removed from the interface and (optionally) deprecated on `PortalInfo` itself

Reply:
Yup!  Adding this to the other commentary for final inclusion